docs/dev/release-process.md

```diff
@@ -0,0 +1,18 @@
+# The Release Process
+
+This project currently follows a [Continuous Delivery](https://en.wikipedia.org/wiki/Continuous_delivery) process, but it's gradually moving toward [Continuous Deployment](https://en.wikipedia.org/wiki/Continuous_deployment).
+
+Whenever a commit is merged to this repository's `main` branch, the deployment pipeline kicks in, deploying the changeset to the [`stage` environment](../firefox.md).
```

Comment:
Nit: I think you can do ```[`stage` environment](../firefox.md#stage)``` to link to the "stage" section.

Reply:
Oh, correct! Good catch